src/Pages/Donate.jsx

```diff
@@ -40,8 +41,21 @@ export const Donate = () => {
           very best avian veterinary care whenver it&apos;s needed, a healthy diet, a clean, safe
           enclosure, regular beak and nail filings, and plenty of enrichment so they can live
           happily while they&apos;re with us at the Refuge.
-          <br />
-          <br />
+          <Typography
+            variant="h2"
+            sx={{ mt: 2, mb: 2 }}
+          >
+            Amazon Wishlist
+          </Typography>
+          The birds went window shopping and picked out some of their favorite things. You can find
+          them on our{" "}
+          <Link
+            href="https://www.amazon.com/hz/wishlist/ls/1YMMJ3VB5032X/ref=hz_ls_biz_ex"
+            target="_blank"
```

Comment:
A couple ideas here: from working on another project, I learned that it can be good practice to add a `rel` attribute on links that open in new tabs and include `"noopener noreferrer"` as the value. These can be helpful security and privacy improvements, as they keep the new tab that opens from having a reference back to the source page on the `window.opener` property.

Second, should we include an `aria-label` for accessibility, to let people using screen readers know what happens if they follow the link?

If you want to make these changes, a revised `<Link>` component could look like this:

```
<Link
    href="https://www.amazon.com/hz/wishlist/ls/1YMMJ3VB5032X/ref=hz_ls_biz_ex"
    target="_blank"
    rel="noopener noreferrer"
    aria-label="Visit the Refuge's Amazon Wish List (opens in a new tab)"
>
    Amazon Wishlist
</Link>
```

Reply:
Fabulous suggestions! ✨ I just looked up `rel="noopener noreferrer"` because I thought it was deprecated, but it looks like it's still relevant for older browsers. I love that you're thinking about accessibility! I'll add your suggestions in now. 🎉